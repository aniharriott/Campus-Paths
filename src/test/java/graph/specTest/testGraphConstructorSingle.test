# Test Graph constructor single file.
# Tests if creating a single graph
# works.

# Create a graph
CreateGraph graph1