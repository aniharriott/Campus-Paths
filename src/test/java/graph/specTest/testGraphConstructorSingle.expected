# Test Graph constructor single file.
# Tests if creating a single graph
# works.

# Create a graph
created graph graph1